### PR TITLE
chore: bump RSDKUtils to 3.0.0, fix imports (SDKCF-5246)

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
* Updated RSDKUtils to 3.0.0
Main benefits: removed bundle assets workaround, precise error message when `toAfterTimeout()` fails in unit tests.
~* Fixed imports in unit tests (unfortunately not all tests cases work properly in SPM mode)~
~* Added missing unit tests file~

EDIT: Moved last two points to #172

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [] I ran `fastlane ci` without errors

### CODE REVIEW

File: Tests/Tests/ViewModelSpec.swift [hunk 1]
```diff
@@ -11,38 +11,31 @@ class ViewModelSpec: QuickSpec {
...
-                it("should return false if bodyMessage, header and messageLowerBody are nil") {
-                    let viewModel = createViewModel(header: nil, messageBody: nil, messageLowerBody: nil)
+                it("should return false if bodyMessage and header") {
+                    let viewModel = createViewModel(header: nil, messageBody: nil)
                     expect(viewModel.hasText).to(beFalse())
                 }
 
-                it("should return false if bodyMessage, header and messageLowerBody are empty") {
-                    let viewModel = createViewModel(header: "", messageBody: "", messageLowerBody: "")
+                it("should return false if bodyMessage and header") {
```
Comment (apparition47) [inline Tests/Tests/ViewModelSpec.swift]: and header..... ?

File: Tests/Tests/ViewModelSpec.swift [hunk 2]
```diff
@@ -11,38 +11,31 @@ class ViewModelSpec: QuickSpec {
...
                 }
 
-                it("should return true if messageLowerBody is not nil") {
-                    let viewModel = createViewModel(header: nil, messageBody: nil, messageLowerBody: "lowerBody")
-                    expect(viewModel.hasText).to(beTrue())
-                }
-
-                it("should return false if bodyMessage, header and messageLowerBody are nil") {
-                    let viewModel = createViewModel(header: nil, messageBody: nil, messageLowerBody: nil)
+                it("should return false if bodyMessage and header") {
```
Comment (apparition47) [inline Tests/Tests/ViewModelSpec.swift]: and header..... ?

**Review by apparition47 — CHANGES_REQUESTED** [review]

File: Tests/Tests/ViewModelSpec.swift (hunk 1 above)
Comment (imaginaris) [reply Tests/Tests/ViewModelSpec.swift]: oops. deleted too much :)

File: Sources/RInAppMessaging/Utilities/BundleInfo.swift [hunk 3]
```diff
@@ -53,10 +53,6 @@ internal extension Bundle {
         module
         #else
```
Comment (imaginaris) [inline Sources/RInAppMessaging/Utilities/BundleInfo.swift:55]: here I could just simply return `assetsBundle(bundleIdPhrase: "RInAppMessagingResources")`. The difference is that the current implementation tries to get resources based on IAM SDK bundle path where the other solution iterates over all available resource bundles. The current approach, to me, seems more precise and safe in terms of name conflicts etc (?). But I'm not sure if that's a good approach

**Review by malayli — APPROVED** [review]

File: fastlane/Fastfile [hunk 4]
```diff
@@ -42,7 +42,10 @@ platform :ios do
   end
 
   lane :build_swift_package do
-    sh 'swift build -Xswiftc "-sdk" -Xswiftc `xcrun --sdk iphonesimulator --show-sdk-path` -Xswiftc "-target" -Xswiftc "x86_64-apple-ios15.0-simulator"'
+    Dir.chdir("..") do
+      ios_version = sh 'xcodebuild -showBuildSettings | grep "IPHONEOS_DEPLOYMENT_TARGET =" | tr -d "IPHONEOS_DEPLOYMENT_TARGET ="'
```
Comment (donnie-jp) [inline fastlane/Fastfile]: 👍

**Review by donnie-jp — APPROVED** [review]

**Review by apparition47 — APPROVED** [review]

**Pull request merged**